cmd/server/flags.go

```diff
@@ -500,11 +524,17 @@ var flags = append([]cli.Flag{
 			cli.EnvVar("WOODPECKER_ENCRYPTION_KEY")),
 		Name:  "encryption-raw-key",
 		Usage: "Raw encryption key",
+		Config: cli.StringConfig{
+			TrimSpace: true,
+		},
 	},
 	&cli.StringFlag{
 		Sources: cli.EnvVars("WOODPECKER_ENCRYPTION_TINK_KEYSET_FILE"),
 		Name:    "encryption-tink-keyset",
 		Usage:   "Google tink AEAD-compatible keyset file to encrypt secrets in DB",
+		Config: cli.StringConfig{
```

Comment:
This one seems to be wrong. It does not load the value from a file.

Reply:
hehe right that sliped in